<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>345</height>
     </rect>
    </property>
    <property name="title">
     <string>Libro:</string>
    </property>
    <widget class="QLineEdit" name="Editorial_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>30</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Titulo:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Autor:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Agregar_Inicio_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>246</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar Inicio</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Titulo_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Publicado:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Agregar_Final_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>202</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar Final</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Editorial:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Autor_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="Publicado_spinbox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2000</number>
     </property>
     <property name="maximum">
      <number>2020</number>
     </property>
    </widget>
    <widget class="QPushButton" name="Mostrar_pushbutton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="Salida">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>271</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
